src/CoreBundle/DependencyInjection/ChameleonSystemCoreExtension.php

```diff
@@ -57,6 +57,10 @@ public function load(array $config, ContainerBuilder $container)
         $this->configureSession($container);
 
         $this->addResources($container);
+
+        $resourceCollectionDefinition = $container->getDefinition('chameleon_system_core.resource_collector');
```

Comment:
Could be done with a method (after configureSession - consistent with those methods there).

Reply:
ok - moved